Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -212,7 +212,7 @@ public extension _CurrentChatUserController {
     /// `setUser` must be called before calling this.
     /// - Parameters:
     ///   - id: Device id to be removed. You can obtain registered devices via `currentUser.devices`.
-    ///   If `currentUser.devices` is not up-to-date, please make an `updateDevices` call.
+    ///   If `currentUser.devices` is not up-to-date, please make an `fetchDevices` call.
```

Comment:
This seems to be outdated, shouldn't it rather be `synchronize`?

Reply:
ping @nuno-vieira

Reply:
@VojtaStavik Fixed it